src/sentry/conf/server.py

```diff
@@ -3981,6 +3981,16 @@ def build_cdc_postgres_init_db_volume(settings: Any) -> dict[str, dict[str, str]
 EVENT_PROCESSING_STORE = "rc_processing_redis"
 COGS_EVENT_STORE_LABEL = "bigtable_nodestore"
 
+# Devserver configuration overrides.
+ngrok_host = os.environ.get("SENTRY_DEVSERVER_NGROK")
+if ngrok_host and SILO_MODE != "REGION":
+    SENTRY_OPTIONS["system.url-prefix"] = f"https://{ngrok_host}"
+    CSRF_TRUSTED_ORIGINS = [f".{ngrok_host}"]
+    ALLOWED_HOSTS = [f".{ngrok_host}", "localhost", "127.0.0.1", "docker.internal"]
```

Comment:
i believe the last one should a leading `.` as well! at least that's what i have in my config